### ISSUE TITLE
When node scaling on post-upgrade cluster that was created on v217 node scaling fails with ectd/CP nodepool

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#19033]
Version: v2.1.7 -> master upgrade

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Start with v2.1.7 rancher and deploy an RKE cluster with nodepool ectd+control plane and nodepool worker (I used digitalocean cluster) each have 1 node.
- Upgrade to master
- Post-upgrade, scale worker nodepool from 1 to 2 nodes. This works fine. Now, scale ectd+CP nodepool from 1 to 2 nodes. This will cause an error.

**Result:**
You will get an error. Pictured is DO cluster having this issue (in that case our nodepool was ALL) and ec2 having this issue (in that case our nodepools were as steps indicate. 1 to 2 worker scale worked, 1 to 2 etcd+CP scale failed)
![image](https://user-images.githubusercontent.com/45179589/54651281-64887300-4a6f-11e9-9175-b9f381584630.png)
And in text:
```
[Failed to create Certificates deployer container on host [206.189.72.97]: Error response from daemon: invalid environment variable: =-----BEGIN CERTIFICATE----- MIIC6DCCAdCgAwIBAgIIHned8Cn0rrQwDQYJKoZIhvcNAQELBQAwEjEQMA4GA1UE AxMHa3ViZS1jYTAeFw0xOTAzMjAwMDExNTJaFw0yMDAzMTkwMDExNTRaMC0xFTAT BgNVBAoTDHN5c3RlbTpub2RlczEUMBIGA1UEAxMLc3lzdGVtOm5vZGUwggEiMA0G CSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDteo6xlYBhOp8DecLHBBIGD1TBXTa9 l6H45GzjDXLgY2TcfZuSgisIMb2/jYe3xK+ITaKd9Kk4FbehZfuTUHeDDitieCdn +c6s8OqD4H1QRAZNuY7/bXoaU1RAwkFj2fMb62kxzXDg/le9HNRdL0MyLP7e7R5P bqxzZDzp/+GBlUWC+kpfZvuElxvTYbEgs+iwLiFyGXeurww84gfn5OJ5tIra/9rf q2V4ar3mQ96pU2NJ/SyJk8d3p02plfWNREF0RsoGkuDwpCUEMSObJrU2/SQQdWsz +jnNr5pMUJ7a0iruetWjE+QCXQI+ZrLgWbtI05l2YPRXpnUYWg3fByy1AgMBAAGj JzAlMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDAjANBgkqhkiG 9w0BAQsFAAOCAQEAsfEBi3i3xtTSm+MxsKM2TASd/LhyXAGxmG+7IpD4f4x7gaOd +3cYNSYWvAcomvHYDZo3i/P+2oGa4i08r/7R06iZciQkIEwc9U3uOddW0ck1gtp8 NbYz4lnxG1SZje+NN40UiRb5h/CUROhuFXHgplS/hTzu2yr368f7I1gsTtZ4H92m t4QkP4Cb2V4HPjCSBsI06OS82SZSyYHRaLXZ+B8mpnAl//8LswGxixB2CjckTvcf P99UsMMdhGfLnb7qejItjpr6DUzP37xcFu6CnuqmzILq6dRwZ/Ue4ZMBs7T2dzNe ic2hMqEPR4oH4/5xf5UDMBUS96lAr0vI84bqYw== -----END CERTIFICATE-----]
```

**Other details that may be helpful:**
We tried with DO / EC2 and replicated in both. Azure likely has the problem too but we just saw generic "ectd is not healthy" message for it when we did this.

**Comment by sangeethah** [comment]
Tested the following scenarios;
Single node rancher server setup on v2.1.7 with following DO clusters:
1. cluster1 -  1 node (all roles)
2. cluster2 -  1 control node , 1 etcd node and 1 worker node.

Rancher Server upgraded to  mater builds - version 68e56010.

1. Edit cluster1 and scale up the existing node pool to 2. 
Cluster gets to "Active" state with 2 nodes successfully.
Edit cluster1 and scale up the existing node pool to 3.
Cluster gets to "Active" state with 3 nodes successfully.

2. Edit cluster2 and scale up etcd node pool to 2. 
Cluster gets to "Active" state with 2 etcd nodes.
 Edit cluster2 and scale up worker node pool to 2. 
Cluster gets to "Active" state.
 Edit cluster2 and scale up etcd node pool to 3.
Cluster gets to "Active" state.
 Edit cluster2 and scale up control node pool to 2.
Cluster gets to "Active" state.

Note- After each cluster update , when cluster got to "Active" state , able to scale up existing workloads and check that ingress is able to reach the newly scaled up instances.

**Comment by davidnuzik** [comment]
Version: v2.1.7 -> master upgrade

Tested again, following my steps to reproduce. In each case I was able to successfully scale up without any issues. I also created a couple of clusters with more unique node pools. Each scaled up without any issues. 

Some examples:
Azure RKE cluster:
1 nodepool of etcd+cp
1 nodepool of worker
- Can scale both up to 2 (each) without any issues

Amazon RKE cluster:
1 nodepool of ectd
1 nodepool of cp
1 nodepool of wokrer
- can scale up each to 2 without any issues

Digital Ocean RKE cluster:
1 nodepool of ALL
- can scale up to 2 without any issues
